### PR TITLE
Create ez_35.mdx

### LINKED ISSUE #92: Create a page for `ez_35`
### description
Make `/docs/upgrades/locks/ez_35.mdx` using the [lock template](https://github.com/comcode-org/hackmud_wiki/blob/main/templates/lock.mdx) as a starter. Write a page about the lock that complies with the [content guide](https://github.com/comcode-org/hackmud_wiki?tab=readme-ov-file#content-guide).

### PR DESCRIPTION
**Pull request by kbinreallife**
fixes #92

### CODE REVIEW

File: docs/upgrades/locks/ez_35.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ez_35 has no additional stats. The upgrade only spawns at 'noob' rarity.
+
+## Behavior
+
+When encountered, the lock expects a string as its first parameter, and a digit as its second. Each parameter will return corresponding errors if an incorrect data type is input.
+
+## Solving
+
+<details>
+  <summary>Spoilers for ez_35's solutions.</summary>
```
Comment (Fayti1703) [inline docs/upgrades/locks/ez_35.mdx:20]: This summary doesn't appear properly. (Unfortunately I don't know why)

**Review by Fayti1703 — CHANGES_REQUESTED** [review]

File: docs/upgrades/locks/ez_35.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  The first parameter's possible strings are, "open", "unlock", and "release". The
+  second parameter is a single digit, input as a number, ranging from 0 to 9.
```
Comment (Fayti1703) [inline docs/upgrades/locks/ez_35.mdx:22]: Add some newlines here to break the paragraph correctly, which fixes the summary thing

```suggestion
  <summary>Spoilers for ez_35's solutions.</summary>
  
  The first parameter's possible strings are, "open", "unlock", and "release". The
  second parameter is a single digit, input as a number, ranging from 0 to 9.
  
```

File: docs/upgrades/locks/ez_35.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+</details>
+
+## Example unlock
+
+<details>
+<summary>Spoilers for ez_35's solutions</summary>
+```
+>>user.fakeloc123 \{ ez_35: "open", digit: 0 }
```
Comment (Fayti1703) [inline docs/upgrades/locks/ez_35.mdx:30]: Remove the backslash here, it shows up in the output.
```suggestion
>>user.fakeloc123 { ez_35: "open", digit: 0 }
```

**Review by Fayti1703 — CHANGES_REQUESTED** [review]

**Comment by kbinreallife** [comment]
prettier is sometimes adding > > where they don't need to be, and is sometimes spacing out ones that exist. it is also adding additional triple (and in some silly cases, quadruple) backticks at the end of the page.

**Review by Fayti1703 — APPROVED** [review]

**Pull request merged**